tox.ini

```diff
@@ -18,69 +18,69 @@ python =
     3.12: py312
     3.13: py313, lint, docs, pyright
 
-[pytest]
-pythonpath = . tests
-DJANGO_SETTINGS_MODULE = django_minio_storage_tests.settings
-addopts=--tb=short
-python_files = tests.py test_*.py *_tests.py
-django_find_project = false
-
 [testenv]
-commands = pytest {posargs}
-setenv =
-    PYTHONDONTWRITEBYTECODE=1
-    MINIO_STORAGE_ENDPOINT={env:MINIO_STORAGE_ENDPOINT:localhost:9153}
-    MINIO_STORAGE_ACCESS_KEY={env:MINIO_STORAGE_ACCESS_KEY:weak_access_key}
-    MINIO_STORAGE_SECRET_KEY={env:MINIO_STORAGE_SECRET_KEY:weak_secret_key}
-    TOX_ENVNAME={envname}
+package = wheel
+set_env =
+    PYTHONDONTWRITEBYTECODE = 1
+    MINIO_STORAGE_ENDPOINT = {env:MINIO_STORAGE_ENDPOINT:localhost:9153}
+    MINIO_STORAGE_ACCESS_KEY = {env:MINIO_STORAGE_ACCESS_KEY:weak_access_key}
+    MINIO_STORAGE_SECRET_KEY = {env:MINIO_STORAGE_SECRET_KEY:weak_secret_key}
+    TOX_ENVNAME = {envname}
 deps =
     django42: Django==4.2.*
     django51: Django==5.1.*
     django52: Django==5.2.*
     minio: minio
     minioknown: minio==7.1.12
     -rdev-requirements.txt
+commands =
+    pytest {posargs}
 
 [testenv:py313-django52-minioknown]
-commands = pytest --cov --cov-append --cov-report=term-missing {posargs}
+commands =
+    pytest --cov --cov-append --cov-report=term-missing {posargs}
 
 [testenv:coverage-report]
-deps = coverage[toml]
-skip_install = true
+package = skip
+depends = py313-django52-minioknown
+deps =
+    coverage[toml]
 commands =
     coverage report
     coverage html
-depends=py313-django52-minioknown
 
 [testenv:pyright]
+depends = py313-django52-minio
```

Comment:
This uses the latest supported Django for type correctness. Ideally we want to also support the oldest Django, but `django-stubs` is imperfect and tends to improve with each new release. Also, Django is very conservative in its API changes, so new versions shouldn't be outright incompatible with existing supported versions.